src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -1852,15 +1980,26 @@ void M(Action<int, int> a)
     {
         M((x[||]
     }
-}");
+}";
+            await VerifyCS.VerifyRefactoringAsync(code,
+                new[]
+                {
+                    // /0/Test0.cs(8,12): error CS0103: The name 'x' does not exist in the current context
+                    DiagnosticResult.CompilerError("CS0103").WithSpan(8, 12, 8, 13).WithArguments("x"),
+                    // /0/Test0.cs(8,13): error CS1002: ; expected
+                    DiagnosticResult.CompilerError("CS1002").WithSpan(8, 13, 8, 13),
+                    // /0/Test0.cs(8,13): error CS1026: ) expected
+                    DiagnosticResult.CompilerError("CS1026").WithSpan(8, 13, 8, 13),
+                    // /0/Test0.cs(8,13): error CS1026: ) expected
+                    DiagnosticResult.CompilerError("CS1026").WithSpan(8, 13, 8, 13),
```

Comment:
📝 These should all be specified either directly in the test input, or using `WithLocation(int)` instead of `WithSpan`.

Reply:
I thought the idea of the test output was to make it easy to copy and paste into tests? If that is true, then the test output should probably be updated.